setup.py

```diff
@@ -115,30 +116,40 @@ def user_dir():
 
 class DevelopCmd(develop):
     prefix_targets = [
-        ("nbconvert/templates", 'jdaviz-default'),
-        ("voila/templates", 'jdaviz-default')
+        (os.path.join("nbconvert", "templates"), 'jdaviz-default'),
+        (os.path.join("voila", "templates"), 'jdaviz-default')
```

Comment:
```suggestion
        (Path("nbconvert", "templates"), 'jdaviz-default'),
        (Path("voila", "templates"), 'jdaviz-default')
```

Reply:
Your suggestions also change the Unix behaviors. I'll have to think if this is worth the extra time... Feel free to PR against this PR with a working solution if you get to it first. I am working on other stuff right now. Thanks for the review!

Reply:
> Feel free to PR against this PR with a working solution if you get to it first

I was quietly hoping for the green light 😁! Hopefully I'll be able to carve out some time soon!

Reply:
Lemme see if I can push your idea tonight to a working state. Will get back to you.